src/main/java/gyro/google/iam/ServiceAccountResource.java

```diff
@@ -0,0 +1,254 @@
+    private String name;
+    private Boolean enableAccount;
+
+    // Read-only
+    private String id;
+    private String email;
+
+    /**
+     * The friendly display name for the service account. (Required)
+     */
+    @Updatable
+    @Required
+    public String getDisplayName() {
+        return displayName;
+    }
+
+    public void setDisplayName(String displayName) {
+        this.displayName = displayName;
+    }
+
+    /**
+     * The optional description of the service account.
+     */
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    /**
+     * The unique name of the service account. It can be a string starting with a lowercase letter followed by lower case alphanumerical characters or hyphens and between 6 and 30 characters. (Required)
+     */
+    @Required
+    @Regex(value = "^[a-z]([a-z]|[0-9]|-){5,29}$", message = "a string starting with a lowercase letter followed by lower case alphanumerical characters or hyphens and between 6 and 30 characters.")
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    /**
+     * Enable or disable the service account.
+     */
+    @Updatable
+    public Boolean getEnableAccount() {
+        return enableAccount;
+    }
+
+    public void setEnableAccount(Boolean enableAccount) {
+        this.enableAccount = enableAccount;
+    }
+
+    /**
+     * The ID of the service account.
+     */
+    @Id
+    @Output
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    /**
+     * The email address of the service account.
+     */
+    @Output
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    @Override
+    public void copyFrom(ServiceAccount model) throws Exception {
+        setId(model.getName());
+        setEmail(model.getEmail());
+        setName(Utils.getServiceAccountNameFromId(model.getName()));
+        setDescription(model.getDescription());
+        setDisplayName(model.getDisplayName());
+        setEnableAccount((model.getDisabled() == null || model.getDisabled().equals(Boolean.FALSE)) ? Boolean.TRUE : Boolean.FALSE);
+    }
+
+    @Override
+    public String primaryKey() {
+        return super.primaryKey();
+    }
+
+    @Override
+    protected boolean doRefresh() throws Exception {
+        Iam client = createClient(Iam.class);
+
+        List<ServiceAccount> serviceAccounts = client.projects()
+            .serviceAccounts()
+            .list("projects/" + getProjectId())
+            .execute()
+            .getAccounts()
+            .stream()
+            .filter(r -> r.getName().equals(getId()))
+            .collect(
+                Collectors.toList());
```

Comment:
If you only care about one, then `findFirst()` is the way to go.
```suggestion
        ServiceAccount serviceAccount = client.projects()
            .serviceAccounts()
            .list("projects/" + getProjectId())
            .execute()
            .getAccounts()
            .stream()
            .filter(r -> r.getName().equals(getId()))
            .findfirst()
            .orElse(null);
```